src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -761,6 +760,13 @@ public override void DeleteGeometryForIdentifier(string identifier, bool request
             DeleteGeometries(geometryModels, requestUpdate); 
         }
 
+        protected override void OnRenderPackagesUpdated(NodeModel node, IEnumerable<IRenderPackage> packages)
```

Comment:
I am fine with either approach. Moving the code to base class makes code cleaner but shuts the door completely. What if we want to customize a watch node monitoring the moving path of a point, the input keep changing and in this case we do not want to remove geometry in base class right? Maybe it's a silly idea :)